ui/component/or-dashboard-builder/src/or-dashboard-widgetcontainer.ts

```diff
@@ -67,7 +73,7 @@ export class OrDashboardWidgetContainer extends LitElement {
             const oldVal = changedProps.get('widget') as DashboardWidget | undefined;
             const idChanged = oldVal?.id !== this.widget?.id;
             const nameChanged = oldVal?.displayName !== this.widget?.displayName;
-            const configChanged = JSON.stringify(oldVal?.widgetConfig) !== JSON.stringify(this.widget?.widgetConfig);
+            const configChanged = JSON.stringify(oldVal?.widgetConfig) !== JSON.stringify(this.getWidgetConfig());
```

Comment:
stringify comparison is not reliable for objects as key order is not guaranteed, that's what `Util.objectsEqual()` method is for

Reply:
Agreed. I adjusted it in the newest commit.